great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -106,6 +108,10 @@ def run_results(
     def run_id(self) -> RunIdentifier:
         return self._run_id
 
+    @property
+    def validation_result_url(self) -> str:
+        return self._validation_result_url
```

Comment:
This should be `Optional[str]` I think

Reply:
We aren't type-checking this code otherwise CI would catch it.